rules/codelingo/code-review-comments/go-error-fmt/codelingo.yaml

```diff
@@ -62,24 +62,48 @@ funcs:
 
         return surroundWithQuotes(escapeGoFormat(uncapitalizeFirstLetter(removeTrailingPunctuation(text))))
       }
-tenets:
+rules:
   - name: go-error-fmt
     actions:
-      codelingo/rewrite:
+      codelingo/review:
+        comment: Error strings should not be capitalized (unless beginning with proper nouns 
+          or acronyms) or end with punctuation, since they are usually printed following
+          other context.
       codelingo/docs:
         title: Go Error Format
         body: |
           Error strings should not be capitalized (unless beginning with proper nouns 
-          or acronyms) or end with punctuation, since they are usually printed following
-          other context. That is, use fmt.Errorf("something bad") not fmt.Errorf("Something bad"),
-          so that log.Printf("Reading %s: %v", filename, err) formats without a spurious 
-          capital letter mid-message. This does not apply to logging, which is implicitly
-          line-oriented and not combined inside other messages.
-      codelingo/review:
-        comment: |
-          Error strings should not be capitalized (unless beginning with proper nouns 
-          or acronyms) or end with punctuation, since they are usually printed following
-          other context.
+          or acronyms) or end with punctuation. For instance, use fmt.Errorf("something bad") 
+          instead of fmt.Errorf("Something bad"), so that log.Printf("Reading %s: %v", filename, err) 
+          formats without a spurious capital letter mid-message. This does not apply to logging, 
+          which is implicitly line-oriented and not combined inside other messages.
+
+          ## Bad code
+          ```golang
+          // ReadConfig opens a file
+          func ReadConfig(path string) error {
```

Comment:
Can you make these examples actually read the file? Or make a simpler example? It's not very realistic to just open a file and close it again.